cypress/e2e/pages/mediaAssetPage/index.cy.js

```diff
@@ -1,11 +1,324 @@
-import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import { testsThatFollowSmokeTestConfigForAMPOnly } from './testsForAMPOnly';
-import { testsThatFollowSmokeTestConfigForCanonicalOnly } from './testsForCanonicalOnly';
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
+import { testsThatAlwaysRunForAllPages as testsForAllPages } from '../testsForAllPages';
+import { testsThatFollowSmokeTestConfigForAllCanonicalPages as testsForAllCanonicalPages } from '../testsForAllCanonicalPages';
+import { testsThatFollowSmokeTestConfigForAllAMPPages as testsForAllAMPPages } from '../testsForAllAMPPages';
```

Comment:
I've just realised that we can use the default import here (because we have used `export default` in those files)

```suggestion
import testsForAllCanonicalPages from '../testsForAllCanonicalPages';
import testsForAllAMPPages from '../testsForAllAMPPages';
```

Reply:
Ignore me, I was looking at the wrong tests! I thought it was the ones within mediaAssetPage, but these are actually at the level above.

I would like to sort that out in future, but only after we've migrated all the tests. Can we add another ticket to the refactor epic?

Reply:
I would, I am just not sure what we are after here. Do we want to always use `default imports` and does that mean that we enforce a rule where modules have only one export (now we have a mix of default export and named exports in the same file).